aws/container-linux/kubernetes/elb.tf

```diff
@@ -7,37 +7,49 @@ resource "aws_route53_record" "apiserver" {
 
   # AWS recommends their special "alias" records for ELBs
   alias {
-    name                   = "${aws_elb.apiserver.dns_name}"
-    zone_id                = "${aws_elb.apiserver.zone_id}"
+    name                   = "${aws_lb.apiserver.dns_name}"
+    zone_id                = "${aws_lb.apiserver.zone_id}"
     evaluate_target_health = true
   }
 }
 
 # Controller Network Load Balancer
-resource "aws_elb" "apiserver" {
-  name            = "${var.cluster_name}-apiserver"
-  subnets         = ["${aws_subnet.public.*.id}"]
-  security_groups = ["${aws_security_group.controller.id}"]
-
-  listener {
-    lb_port           = 443
-    lb_protocol       = "tcp"
-    instance_port     = 443
-    instance_protocol = "tcp"
-  }
+resource "aws_lb" "apiserver" {
+  name               = "${var.cluster_name}-apiserver"
+  load_balancer_type = "network"
+  subnets            = ["${aws_subnet.public.*.id}"]
+}
 
-  instances = ["${aws_instance.controllers.*.id}"]
+resource "aws_lb_target_group" "apiserver" {
+  name     = "${var.cluster_name}-apiserver"
+  port     = 443
+  protocol = "TCP"
+  vpc_id   = "${aws_vpc.network.id}"
 
   # Kubelet HTTP health check
   health_check {
-    target              = "SSL:443"
+    protocol            = "TCP"
     healthy_threshold   = 2
-    unhealthy_threshold = 4
-    timeout             = 5
-    interval            = 6
+    unhealthy_threshold = 2
```

Comment:
Restore the original health check values please. Let's not slip in unrelated changes.